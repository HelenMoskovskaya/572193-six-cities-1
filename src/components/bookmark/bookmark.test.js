import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router';
import BookMark from './bookmark.jsx';


describe(`BookMark`, () => {
  it(`renders component correctly`, () => {
    const tree = renderer.
      create(<MemoryRouter>
        <BookMark
          changeFavorites={jest.fn()}
          isFavorite={true}
          small={false}
          className={``}
        />
      </MemoryRouter>).
      toJSON();

    expect(tree).toMatchSnapshot();
  });
});
