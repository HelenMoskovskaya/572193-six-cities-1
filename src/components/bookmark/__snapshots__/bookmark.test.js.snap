// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookMark renders component correctly 1`] = `
<button
  className=" button place-card__bookmark-button--active"
  onClick={[MockFunction]}
  type="button"
>
  <svg
    className="place-card__bookmark-icon"
    height={33}
    width={31}
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>
  <span
    className="visually-hidden"
  >
    In bookmarks
  </span>
</button>
`;
